copyrightCheck.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# (C) Copyright IBM Corp. 2020
+#
+# SPDX-License-Identifier: Apache-2.0
+
+files=$(find . -name "*.java" -or -name "*.rb" -or -name "*.sh")
```

Comment:
I think you want "*.scala" for this project.

Reply:
For `flink-pipeline-core` I only check for `*.scala` and `*.sh`.  Should I add in `*.java`?